### ISSUE TITLE
Redeploy john-travoltage

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#86]
John Travoltage is almost ready for a redeploy.  There are just a few issues that need to be completed before it is ready for a release candidate.

- [x] https://github.com/phetsims/tasks/issues/309 (New image)
- [x] https://github.com/phetsims/scenery/issues/515 (Canvas renderer for Edge)
- [x] Apply webgl renderer to ElectronNode, renderer should be null or canvas for Firefox in screen view

Once these are completed, an RC can be created for 1.1 release.

**Comment by jessegreenberg** [comment]
> (Canvas renderer for Edge) this was applied to joist in commit

28f510089c0f5312ca58526b10ff063466ebbc5a

**Comment by jessegreenberg** [comment]
New checkbox added.  @jonathanolson suggests setting renderer to webgl on line 72 in ElectronNode, and try removing renderer override in the screen view.  This could potentially improve performance issues related to https://github.com/phetsims/john-travoltage/issues/80

**Comment by jessegreenberg** [comment]
> Apply webgl renderer to ElectronNode, renderer should be null or canvas for Firefox in screen view

Done, will check performance on Windows 8 Firefox tomorrow to verify that https://github.com/phetsims/john-travoltage/issues/80 is still looking good after adding the webgl renderer to the ElectronNode

**Comment by jessegreenberg** [comment]
The old asset for the image included the whole body and the face. Since the new asset is just the face, we will have to paste the new image over the old asset.

**Comment by ariel-phet** [comment]
@jessegreenberg if it would be easier to have the whole body, I think mariah has that, let me know, I can probably get you whatever is easiest.

**Comment by jessegreenberg** [comment]
Thanks @ariel-phet, it was easy enough to apply the face image.  Here is what it looks like in context:
![capture](https://cloud.githubusercontent.com/assets/6396244/13306954/234a2dc8-db25-11e5-829c-4288ae8ba795.PNG)

Does this look good for the redeploy @ariel-phet?

**Comment by jessegreenberg** [comment]
> Apply webgl renderer to ElectronNode,

**Comment by ariel-phet** [comment]
Can you make the hand black and white @jessegreenberg - it seems a bit odd now that the face is black and white

**Comment by jessegreenberg** [comment]
Sure, how is this:

![capture](https://cloud.githubusercontent.com/assets/6396244/13307341/c1962aa2-db27-11e5-91a5-f71e5240d199.PNG)

**Comment by jessegreenberg** [comment]
I removed the webgl renderer on the ElectronNode.  Performance was fine without it, and forcing the canvas renderer was sufficient to get decent performance on Windows 8 Firefox.

**Comment by samreid** [comment]
How does the iPad do without webGL electrons?

**Comment by jessegreenberg** [comment]
After playing around with iPad and Windows 10 Chrome, I am noticing that the ElectronNode animation is better on *both* devices with ```renderer: webgl```, but ```renderer: webgl``` causes dragging of appendages to perform worse on *both* platforms.

**Comment by ariel-phet** [comment]
Ah, yes, zombie John Travoltage :). It looks good @jessegreenberg, and it will do until I work with Mariah to make the more cartoonish version.

**Comment by samreid** [comment]
> but renderer: webgl causes dragging of appendages to perform worse on both platforms.

It may be because there are too many WebGL layers. AppendageNode has the border highlight as well as the appendage image, so it is properly interlaying svg/webgl/svg/webgl.  With another webgl layer for electrons, it may be too many on iPad.  To test if this is the problem, you could try commenting out the appendagenode borders, and minimizing the number of webgl canvases to see if that helps.

**Comment by jessegreenberg** [comment]
Thanks for the recommendation @samreid.  I just tried removing the borders around the appendages, it did not seem to have an impact on performance for either platform.

At this point I am inclined to remove the ```webgl renderer``` preference from ElectronNode.  ElectronNode animation is not quite as crisp, but overall the performance is better without it.  Animation is still very smooth, even on iPad.  But I would be happy to pursue this further if others think it is necessary.  @samreid and @ariel-phet would you like to investigate this further or should we proceed with an RC for the redeploy?

**Comment by jonathanolson** [comment]
Also let me know if I should take a quick look at what's going on (particularly why WebGL would make some things slower).

**Comment by ariel-phet** [comment]
@jessegreenberg under normal circumstances I would say investigate further...BUT since we are mainly just supposed to be addressing the internationalization issues, please have an issue that summarizes cleanly what you have tried, and leave as a "priority:deferred" issue and move on to RC.

**Comment by jessegreenberg** [comment]
Sounds good @ariel-phet, will create a new issue and prepare the RC.

**Comment by jessegreenberg** [comment]
Wave on a String 1.1 and Ohm's Law 1.3 have different SHAs for 7 common dependencies.  If we are going to take 'stable' common shas from Wave on a String (see https://github.com/phetsims/tasks/issues/476) we should wait to deploy an RC for John Travoltage 1.1 until Wave on a String 1.1 passes through the RC test successfully.

**Comment by jessegreenberg** [comment]
Wave on a String 1.1.0 was recently published and those SHAs passed through testing successfully.  We are ready for a RC of John Travoltage 1.2.0, grabbing the common code SHAs from Wave on a String 1.1.0.

**Comment by jessegreenberg** [comment]
RC created, testing tracked in https://github.com/phetsims/tasks/issues/537.

**Comment by jessegreenberg** [comment]
RC for redeploy passed through testing successfully!  We are ready to redeploy John Travoltage for S2015R.

**Comment by jessegreenberg** [comment]
John Travoltage 1.2.0 has been redeployed! Closing.
